linkis-engineconn-plugins/engineconn-plugins/shell/src/main/scala/org/apache/linkis/manager/engineplugin/shell/executor/ShellEngineConnExecutor.scala

```diff
@@ -77,6 +77,7 @@ class ShellEngineConnExecutor(id: Int) extends ComputationExecutor with Logging
       }) {
         info(line)
         engineExecutionContext.appendTextResultSet(line)
+        engineExecutionContext.appendStdout(line)
```

Comment:
In order to reduce the number of messages, it is recommended to modify it to push by log.`LogHelper.logCache.cacheLog(line)`